src/ApplicationManager.cpp

```diff
@@ -46,17 +49,23 @@ void ApplicationManager::start() {
 /// Run everything with regard to behaviour trees
 void ApplicationManager::runOneLoopCycle() {
     if (io::io.hasReceivedGeom) {
+        if (!fieldInitialized) rtt_success("Received a field message!")
+        fieldInitialized = true;
+
         auto fieldMessage = io::io.getGeometryData().field();
         auto worldMessage = io::io.getWorldState();
 
         if (!SETTINGS.isLeft()) {
-         //   roboteam_utils::rotate(&fieldMessage);
             roboteam_utils::rotate(&worldMessage);
         }
 
         world->updateWorld(fieldMessage, worldMessage); // this one needs to be removed
 
         if (!world->getUs().empty()) {
+            if (!robotsInitialized) {
+                rtt_success("Received robots, starting behaviour trees!")
```

Comment:
```suggestion
                rtt_success("Received first robots, starting behaviour trees!")
```

Reply:
No. Sometimes we lose all robots out of the field and then you can get this message again when they re appear :-)

Reply:
Oh okay, nevermind then <3